codec_test.go

```diff
@@ -66,4 +66,13 @@ func TestCodec(t *testing.T) {
 
 	c = astiav.FindDecoderByName("invalid")
 	require.Nil(t, c)
+
+	var decoders []*astiav.Codec
```

Comment:
I'd replace all this with
```go
var found bool
for _, c := range astiav.Codecs() {
  if c.ID() == astiav.CodecIDMjpeg {
    found = true
  }
}
require.True(t, found)
```

I'd even be tempted to test the whole content of the slice, but I'm not sure windows, linux and macosx will return the same list 🤔  I'll check it out when this PR will be merged 👍

Reply:
Thats true it depense on ffmpeg build it could be that ffmpeg is build without specific Codecs, I See that u use mjpeg for Testing, so thats was the reason why i did this to. We also could use h246 i think most builds would have this.